source/backend/api/Services/TakeService.cs

```diff
@@ -86,20 +86,18 @@ public IEnumerable<PimsTake> UpdateAcquisitionPropertyTakes(long acquisitionFile
             else
             {
                 // Complete Takes can only be deleted or set to InProgress by Admins when File is Active/Draft
-                if (!_user.HasPermission(Permissions.SystemAdmin))
-                {
-                    var currentCompleteTakes = currentFilePropertyTakes
-                        .Where(x => x.TakeStatusTypeCode == AcquisitionTakeStatusTypes.COMPLETE.ToString()).ToList();
+                var currentCompleteTakes = currentFilePropertyTakes
+                    .Where(x => x.TakeStatusTypeCode == AcquisitionTakeStatusTypes.COMPLETE.ToString()).ToList();
 
-                    if (currentCompleteTakes.Count > 0)
+                if (currentCompleteTakes.Count > 0)
+                {
+                    foreach (var completeTake in currentCompleteTakes)
                     {
-                        foreach (var completeTake in currentCompleteTakes)
+                        // Validate that the current completed take can only by updated by a sysadmin
+                        var updatedTake = takes.FirstOrDefault(x => x.TakeId == completeTake.TakeId);
+                        if (!_user.HasPermission(Permissions.SystemAdmin) && (updatedTake is null || (updatedTake is not null && updatedTake.TakeStatusTypeCode != completeTake.TakeStatusTypeCode)))
                         {
-                            var updatedTake = takes.FirstOrDefault(x => x.TakeId == completeTake.TakeId);
-                            if (updatedTake is null || updatedTake?.TakeStatusTypeCode != completeTake.TakeStatusTypeCode)
-                            {
-                                throw new BusinessRuleViolationException("Retired records are referenced for historical purposes only and cannot be edited or deleted. If the take has been added in error, contact your system administrator to re-open the file, which will allow take deletion.");
-                            }
+                            throw new BusinessRuleViolationException("Retired records are referenced for historical purposes only and cannot be edited or deleted. If the take has been added in error, contact your system administrator to re-open the file, which will allow take deletion.");
```

Comment:
This error seems to refer to a take on a retired property, but I don't see anything in the logic above about a retired property. Are you confident this is correct?

Reply:
This came from the merge from another story. I agree, that it does not look like it actually validates for retired.

Reply:
Also the wording of this makes me suspect that this is coming from the acquisition file being completed, but the retired part seems off. I would rather open a new ticket and go over the validation to make sure this is handled correclty taken into accounts all stories related to it